rl4co/envs/common/utils.py

```diff
@@ -44,3 +45,56 @@ def _getstate_env(self):
     state = self.__dict__.copy()
     del state["rng"]
     return state
+
+
+class Generator():
+    def __init__(self, **kwargs):
+        self.kwargs = kwargs
+
+    def __call__(self, batch_size) -> TensorDict:
+        batch_size = [batch_size] if isinstance(batch_size, int) else batch_size
+        return self._generate(batch_size)
+
+    def _generate(self, batch_size, **kwargs) -> TensorDict:
+        raise NotImplementedError
+
+
+def get_sampler(
+        val_name: str,
+        distribution: Union[int, float, str, type, Callable],
```

Comment:
What does this mean, allowing type `type`here? I it to not have to list `Uniform`, `Normal`, etc. individually?

Reply:
This looks a bit wired, but this comes from the type of a `torch.distributions` class, i.e., `print(type(torch.distributions.Uniform))` is `type.`  

The reason why I put `type` here instead of a list of `Union, Normal, ...` is that in `torch.distributions` there are quite a few other distributions, e.g. `Gamma, Beta, ...`, all these are `type`, and at here they will go to the `Callable`. Tbh, only haveing the `Callable` would be enough, but just in case a small hint for a `torch.distributions` case.